discovery_climate_music_analysis/utils/io.py

```diff
@@ -0,0 +1,11 @@
+from discovery_climate_music_analysis import PROJECT_DIR, logging
+from yaml import safe_load
+
+DEF_CONFIG_PATH = PROJECT_DIR / "discovery_climate_music_analysis/config"
+
+
+def import_config(filename: str, path=DEF_CONFIG_PATH) -> dict:
```

Comment:
Hey, it'd be good to specify - either in the docstring here or in a readme file - what you expect to have in this config yaml file, and what format it should follow. I guess in this case it would be the `client_id` and `client_secret` variables.